packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx

```diff
@@ -1,21 +1,30 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useCallback } from 'react';
 import HeaderLogo from '@dailydotdev/shared/src/components/layout/HeaderLogo';
 import {
   Button,
   ButtonVariant,
 } from '@dailydotdev/shared/src/components/buttons/Button';
 import { ArrowIcon } from '@dailydotdev/shared/src/components/icons';
 import { useRouter } from 'next/router';
+import { webappUrl } from '@dailydotdev/shared/src/lib/constants';
 
 export const PlusHeader = (): ReactElement => {
-  const { back } = useRouter();
+  const { back, replace } = useRouter();
```

Comment:
Technically could be something non daily dev right?
Maybe we should somehow check if jt was external?

Reply:
I guess for now it's fine maybe.
More of a nice to have

Reply:
> I guess for now it's fine maybe. More of a nice to have

Yeah, would be nice to have. But probably need to add a global state to track our internal previous page

Reply:
Testing https://as-728-fix-back-button.preview.app.daily.dev/plus

Reply:
Testing2 https://as-728-fix-back-button.preview.app.daily.dev/